GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -1095,6 +1097,7 @@ - (void)OAuthLoginWithOptions:(GIDSignInInternalOptions *)options
   if (keychainError) {
     _saveAuthorizationReturnValue = NO;
   } else {
+    [[[_authState expect] andReturn:tokenResponse] lastTokenResponse];
```

Comment:
Lets add some new tests for the addScopes codepath.

Reply:
Added the test in GIDSignInTest.m